modules/sdk/project-templates/project-templates-api/src/main/resources/archetype-resources/pom.xml

```diff
@@ -11,13 +11,14 @@
 	<version>${version}</version>
 	<profiles>
 		<profile>
+			<id>standalone</id>
 
-			<!-- Enable this profile in standalone -->
+			<!-- Uncomment the following element if project is not in Workspace
```

Comment:
Please update to " ... if the project is not in workspace ... "

Reply:
Okay. I'll resend